pkg/cmd/grafana-cli/commands/cuetsify_command.go

```diff
@@ -27,12 +27,38 @@ import (
 
 var ctx = cuecontext.New()
 
+// The only import statement we currently allow in any models.cue file
 const allowedImport = "github.com/grafana/grafana/packages/grafana-schema/src/schema"
 
 var importMap = map[string]string{
 	allowedImport: "@grafana/schema",
 }
 
+// Hard-coded list of paths to skip. Uncomment a particular file as we're ready
```

Comment:
`Remove a particular file...` or `Comment out a particular file...`?
```suggestion
// Hard-coded list of paths to skip. Remove a particular file as we're ready
```